backend/tests/expenses/test_expense_service.py

```
@@ -1602,53 +1602,45 @@ async def test_get_friends_balance_summary_success(expense_service):
+    #   Group Alpha: Main owes Friend1 50 (balance: -50 for Main)
+    #   Group Beta: Friend1 owes Main 30 (balance: +30 for Main)
+    #   Total for Friend1: -50 + 30 = -20 (Main owes Friend1 20)
     # Friend 2:
-    #   Group Beta: Main owes Friend2 70 (net -70 for Main)
-    #   Total for Friend2: Main owes 70 to Friend2.
+    #   Group Beta: Main owes Friend2 70 (balance: -70 for Main)
+    #   Total for Friend2: -70 (Main owes Friend2 70)
 
-    # This is the side_effect for the .aggregate() call. It must be a sync function
-    # that returns a cursor mock (AsyncMock).
     def sync_mock_settlements_aggregate_cursor_factory(pipeline, *args, **kwargs):
```

Ruff (0.14.7)

Missing return type annotation for private function `sync_mock_settlements_aggregate_cursor_factory`

(ANN202)